backend/src/main/java/com/exadel/carinsurance/service/implementation/AssignmentService.java

```diff
@@ -79,6 +88,39 @@ public ResponseEntity createAssignment( @RequestBody AssignmentRequestEntity req
             );
 
     //  Vehicle condition info creation
+    String namesOfPhotosOfImpact = "";
+
+    try {
+      String directoryPath = String
+          .format(
+              "photosofimpact/userId=%s/assignmentId=%s",

```

Comment:
as this is a static format please extract it as static final property or as configurable parameter using app.properties

Reply:
Done